zerver/views/webhooks/taiga.py

```diff
@@ -0,0 +1,231 @@
+from __future__ import absolute_import
+from zerver.models import get_client
+from zerver.lib.actions import check_send_message
+from zerver.lib.response import json_success, json_error
+from zerver.decorator import REQ, has_request_variables, api_key_only_webhook_view
+
+import ujson
+from six.moves import range
+
+
+@api_key_only_webhook_view
+@has_request_variables
+def api_taiga_webhook(request, user_profile, stream=REQ(default='taiga'), topic=REQ(default='General')):
+
+    templates = {
```

Comment:
I think it'd be cleaner if you moved these templates and functions out of the `api_taiga_webhook` function (they should still be in the file)